### ISSUE TITLE
Accessibility - text in footer has low contrast

### ISSUE THREAD

**Issue by edent** [openjournals/joss#1533]
The text in the footer does not meet WCAG accessibility guidelines. This makes it hard to read.

<img width="1700" height="1029" alt="Text with low contrast." src="https://github.com/user-attachments/assets/7e5d0104-aa1f-418f-8fe4-00e751f8e836" />

The relevant CSS is:

https://github.com/openjournals/joss/blob/60ec700a4863ccc58d037add39e4216ed6d5e2b4/app/assets/stylesheets/application.scss#L371-L379

Assuming you want to keep roughly the same colour scheme, may I suggest setting the text colour to `#D2C2FF` and the link colour to `#8cbcf0`. The hover colour for links should also be changed.

Would you like a PR with these changes?

**Comment by xuanxu** [comment]
Changes applied.
@edent Thanks for the suggestion!
